perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -109,6 +109,15 @@ sub open311_config_updates {
     $params->{mark_reopen} = 1;
 }
 
+sub filter_report_description {
+    my ($self, $description) = @_;
+
+    $description =~ s/\b[\w]+\@[\w]+\.[^ ]+\b//gms;
```

Comment:
The bit before an `@` could also have a variety of characters most notably `.`, but also any of `!#$%&'*+-/=?^_`{|}~`. The bit after can also have hyphen and cannot have underscore (included in `\w`).

Reply:
Not sure what the `m` and `s` modifiers are for here.